### PR TITLE
2670 - Fix calculated height with charts

### LINKED ISSUE #2670: Homepage: chart height is not calculated properly when other elements are used in widget
**Describe the bug**
Homepage: chart height is not calculated properly when other elements are used in widget

**To Reproduce**
1. Add some banner (div) to the widget with chart https://design.infor.com/code/ids-enterprise/latest/demo/components/homepage/example-filled
2. Observe that the height of the chart is still 288px like there is nothing else.

**Expected behavior**
The chart should fit into smaller height.

**Version**
- ids-enterprise: 4.20.0

**Screenshots**
image
image

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed height was not calculating properly when used other elements along content container for bar, column, pie and line with Charts.

**Related github/jira issue (required)**:
Closes #2670

**Steps necessary to review your pull request (required)**:
http://localhost:4000/components/bar-stacked/test-fit-height.html
http://localhost:4000/components/column-stacked/test-fit-height.html
http://localhost:4000/components/line/test-fit-height.html
http://localhost:4000/components/donut/test-fit-height.html
- Pull this branch and build/run the demo app
- Open above links
- See the chart should fit properly and no scroll bar in widget

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -1264,4 +1264,17 @@ utils.isInViewport = function isInViewport(element) {
   );
 };
 
+/**
+ * Get parent available height for given element.
+ * @privateel
+ * @param {object} el The element to get parent available height.
+ * @returns {number} The calculated height.
+ */
+utils.getParentAvailHeight = function getParentAvailHeight(el) {
```
Comment (tmcconechy) [inline src/utils/utils.js:1273]: Maybe lets go with the longer version

```
utils.getParentAvailableHeight = function getParentAvailableHeight(el) {
```

**Comment by deep7102** [comment]
Updated now, and it is ready to review.

**Comment by EdwardCoyle** [comment]
@hohnzy are you able to double-check this branch in your dev environment and let us know if it works for you?

**Comment by EdwardCoyle** [comment]
@deep7102 Seems like this is mostly working well, but I'm still seeing scrolling within the widget on http://localhost:4000/components/line/test-fit-height.html... is that intentional?

**Pull request merged**